vizro-core/docs/pages/user-guides/custom-components.md

```diff
@@ -422,12 +422,12 @@ As mentioned above, custom components can trigger action. To enable the custom c
 
 
         # 1. Create new custom component
-        class Carussel(vm.VizroBaseModel):
-            type: Literal["carussel"] = "carussel"
+        class Carousel(vm.VizroBaseModel):
+            type: Literal["carousel"] = "carousel"
             items: List
             actions: List[Action] = []
-
-            _set_actions = _action_validator_factory("active_index")  # (1)!
```

Comment:
This seems to not work at first glance, but taking a look

Reply:
So just trying out with different syntax:
- `{.python}` works while `{.python pycafe-link} does not
- this could indeed be linked to the plugin

@stichbury Should we reach out to Maarten for that one, or are we content to move this to code comments?